pkgdown/_pkgdown.yml

```diff
@@ -0,0 +1,168 @@
+destination: docs
+
+template:
+  params:
+    bootswatch: simplex
+
+home:
+  strip_header: true
+  links:
+  - text: Learn more
+    href: https://rmarkdown.rstudio.com/
+  - text: Read more
```

Comment:
I guess readers may not know what "Learn more" and "Read more" mean (and what the difference is) until they click these links.

Reply:
Yes, open to suggestions there. If you look on the rendered link, it fills in the "_at `<link>`_" text. I could edit the second to be "Read the book _at `<link>`_" or something simple?

<img width="278" alt="Screen Shot 2019-06-25 at 1 20 39 PM" src="https://user-images.githubusercontent.com/12160301/60130463-19f05400-974c-11e9-89e2-bb12e23e7e3e.png">

Reply:
Perhaps "Read the book"? This is a very minor issue to me.